examples/datasource-http-backend/src/plugin.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
+  "type": "datasource",
+  "name": "Datasource Http Backend",
+  "id": "grafana-datasourcehttpbackend-datasource",
+  "metrics": true,
+  "backend": true,
+  "alerting": true,
+  "executable": "gpx_datasource_http_backend",
+  "info": {
+    "description": "",
+    "author": {
+      "name": "Grafana Labs"
+    },
+    "keywords": ["datasource"],
+    "logos": {
+      "small": "img/logo.svg",
+      "large": "img/logo.svg"
+    },
+    "links": [
+      {
+        "name": "Website",
+        "url": "https://github.com/grafana/grafana-starter-datasource"
+      },
+      {
+        "name": "License",
+        "url": "https://github.com/grafana/grafana-starter-datasource/blob/master/LICENSE"
+      }
+    ],
```

Comment:
Probably also related to create-plugin. After reviewing quite a few plugins I often notice that these same links are not changed.

Reply:
Yes. This was added by create-plugin too

Reply:
I created https://github.com/grafana/plugin-tools/issues/129